.github/workflows/cd.yml

```
@@ -62,3 +62,24 @@ jobs:
           installers-regex: '\.exe$'
           delete-previous-version: 'true'
           token: ${{ secrets.WINGET_TOKEN }}
+
+  cargo-publish:
+    name: Publish to crates.io
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          components: clippy
+
+      - name: Publish to crates.io
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        run: cargo publish
```

```suggestion
      - name: Publish to crates.io
        if: matrix.os == 'ubuntu-latest'
        env:
          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
        run: cargo publish
```

Why not just make it a step like the others?

---

If it's a separate job it can run in parallel without waiting on other steps or making other steps wait on it. But I'm not too attached to making it a separate job, so if you prefer it as a step then :+1: from me 🙂 
I'll be happy either way as long as publishing to crates.io is somehow part of the workflow :smiley: